autocomplete/zsh/_heroku

```diff
@@ -10,6 +10,10 @@ _command_id () {
   _arguments "1:commands:($(grep -oe '^[a-zA-Z0-9:_-]\+' $COMMANDS_PATH))"
 }
 
+_apps () {
+  compadd "$@" - $(heroku autocomplete:values --cmd ${words[2]} --flag '--app')
```

Comment:
@RasPhilCo definitely not today / this PR, but at some point it might be good to dump flags that have autocomplete to a file so that we do not have to hardcode completions both here & in the flag.